discord/ui/view.py

```diff
@@ -312,6 +313,21 @@ def clear_items(self) -> None:
         self.children.clear()
         self.__weights.clear()
 
+    def get_item(self, custom_id: str) -> Optional[Item]:
+        """Get an item from the view by its ``custom_id``. Alias for `utils.get(view.children, custom_id=custom_id)`.
+
+        Parameters
+        -----------
+        custom_id: :class:`str`
+            The custom_id of the item to get
```

Comment:
```suggestion
        """Returns an item from the view given its custom ID. Alias for `utils.get(view.children, custom_id=custom_id)`.
        Parameters
        -----------
        custom_id: :class:`str`
            The custom ID of the item to get.
```

Reply:
Given the wording of other `get` functions, I'll update the wording further (though it's not too consistent anyway)